pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sotopia"
-version = "0.1.0-rc.3"
+version = "0.1.0-rc.5"
```

Comment:
I wonder why directly junp from 3 to 5 and skip 4 🤔, typo?
<img width="331" alt="image" src="https://github.com/user-attachments/assets/db33afd3-ec91-4c20-8c9a-652a0419d471">

Reply:
a typo yeah, my bad